packages/typespec-test/test/openai_modular/generated/typespec-ts/src/utils/deserializeUtil.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Not sure if put the deserializeUtil into src/utils folder is correct.

Reply:
I remember in our last review meeting we mentioned that we could put shared components in `common` folder.

Reply:
This is how notification-hub does https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/notificationhubs/notification-hubs/src/utils